packages/runtime/src/components/_internal/ImplWrapper.tsx

```diff
@@ -20,18 +20,24 @@ const _ImplWrapper = React.forwardRef<HTMLDivElement, ImplWrapperProps>((props,
   const handlerMap = useRef(globalHandlerMap.get(c.id)!);
   const eleRef = useRef<HTMLElement>();
   const onRef = (ele: HTMLElement) => {
-    eleMap.set(c.id, ele);
+    // If a component is in module, it should not have mask, so we needn't set it
+    if (!isInModule) {
+      eleMap.set(c.id, ele);
+    }
     hooks?.didDomUpdate && hooks?.didDomUpdate();
   };
 
   useEffect(() => {
-    if (eleRef.current) {
+    // If a component is in module, it should not have mask, so we needn't set it
+    if (eleRef.current && !isInModule) {
```

Comment:
is it possible to infer whether a component is in a module from the application schema?

Reply:
No, because the component schema in module and application are the same.